WordPress/src/test/java/org/wordpress/android/viewmodel/activitylog/ActivityLogViewModelTest.kt

```diff
@@ -139,4 +152,18 @@ class ActivityLogViewModelTest {
             assertEquals(this.site, site)
         }
     }
+
+    private fun initializeActivityList() {
+        val birthday = Calendar.getInstance()
+        birthday.set(1985, 8, 27)
+        activityLogList.add(ActivityLogModel("", "", "", "", "", "",
+                "", true, "",  birthday.time))
+        activityLogList.add(ActivityLogModel("", "", "", "", "", "",
```

Comment:
consider using Kotlin data class `copy` to copy an object with updated parameter

Reply:
Nice tip!